styles/designs/cardboard/parts/_wrappers-components.scss

```diff
@@ -7,3 +7,5 @@ $SectionWrapper--ThirdLevel: empty_wrapper(ThirdLevel, ' > section > section > s
 $DivWrapper--FirstLevel: empty_wrapper(FirstLevel, ' > div');
 $DivWrapper--SecondLevel: empty_wrapper(SecondLevel, ' > div > div');
 $DivWrapper--ThirdLevel: empty_wrapper(ThirdLevel, ' > div > div > div');
+
+$TopSpacing__Container: empty_wrapper(TopSpacingContainer, '');
```

Comment:
I have moved `TopSpacingContainer` to wrappers components in every theme because it is used also by TableTopSpacingShape now.